packages/jest-expo/src/index.js

```diff
@@ -60,40 +59,6 @@ export function unmockAllProperties() {
   allOriginalPropertyDescriptors.clear();
 }
 
-export function mockPlatformIOS() {
```

Comment:
Since this is a breaking change for jest-expo, add an entry to CHANGELOG.md with `[breaking]` so it's easier to remember this for the SDK 36 release notes in a couple months.